packages/app-admin-ui/client/components/Nav/index.js

```diff
@@ -289,8 +292,70 @@ function PrimaryNavItems({
   );
 }
 
-let PrimaryNavContent = ({ mouseIsOverNav }) => {
-  let {
+const UserInfoContainer = styled.div`
+  align-self: stretch;
+  padding: ${PRIMARY_NAV_GUTTER}px 0;
+  margin: 0 ${PRIMARY_NAV_GUTTER}px;
+  border-bottom: 2px solid ${colors.N10};
+  display: flex;
+  align-items: center;
+  font-size: 1.3em;
+`;
+
+const UserIcon = styled.div`
+  flex-shrink: 0;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  background-color: ${colors.primary};
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-right: ${PRIMARY_NAV_GUTTER}px;
+`;
+
+const UserInfo = ({ authListKey, authListPath }) => {
+  // We're assuming the user list as a 'name' field
+  const AUTHED_USER_QUERY = gql`
+    query {
+      user: authenticated${authListKey} {
+        id
+        name
+      }
+    }
+  `;
+
+  // FIXME: figure out how best to handle a case where the user changed their name from the Admin UI.
+  // Perhaps a subscription once those are implemented?
+  const { data: { user } = {}, loading } = useQuery(AUTHED_USER_QUERY);
```

Comment:
Happy to live with this but will ask if anyone knows a good way to clear the apollo cache